### ISSUE TITLE
[D2] 2. Team Feature and Requirements

### ISSUE THREAD

**Issue by AiXin18** [yi-kiat/ICT2106_P3-5#59]
Team Feature and Requirements

List the user features your team is working on, status of completion of the feature (complete or incomplete), and owner. This is also where you should include your Use Case diagram (if it is different from D1).

Highlight any changes from D1 (if any) eg. Feature dropped, feature changed etc

To be completed on - 31/3/2023

**Comment by AiXin18** [comment]
In this section, we will list the features that we have developed and provide an update on their progress. We will use red highlights to indicate features that we have dropped, and green highlights for features that we have added. We will also discuss the functional and non-functional aspects of the features.



![Image](https://user-images.githubusercontent.com/91781056/229053456-2726fa34-e375-4bf6-989b-6c35978f703e.png)


![Image](https://user-images.githubusercontent.com/91781056/229053524-57e0baeb-8b7c-4358-b3ad-8bfbfb7aa73c.png)


![Image](https://user-images.githubusercontent.com/91781056/229053587-ef5f8190-f2e7-49b2-ac07-b0690d1f78f4.png)
